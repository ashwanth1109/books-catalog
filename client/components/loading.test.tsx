import { render } from "@testing-library/react";
import Loading from "./loading.component";

describe("loading.component.tsx", () => {
  it("should assert that dom snapshot is correct", () => {
    const { container } = render(<Loading />);

    expect(container).toMatchSnapshot();
  });
});
