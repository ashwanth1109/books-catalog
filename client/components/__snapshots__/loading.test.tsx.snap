// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading.component.tsx should assert that dom snapshot is correct 1`] = `
<div>
  <div>
    <h1>
      Loading ...
    </h1>
  </div>
</div>
`;
